### ISSUE TITLE
Articulations appearance offset is doubled when applying to the note and the articulation itself

### ISSUE THREAD

**Issue by abariska** [musescore/MuseScore#16694]
### Issue type

Engraving bug

### Bug description

As I see the problem is that appearance offset is applied to articulation when it changed for its note.
Also reproduces with accidentals when applying appearance offset to whole selected measure

### Steps to reproduce

1. Create a simple score with few notes
2. Add an accent and accidental to a note
3. In Appearance window change Offset

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/101250347/223568377-e49e2429-d40b-4872-a0b7-147027ae4b21.mp4

### MuseScore Version

MuseScore 4.0.2 

### Regression

No.

### Operating system

macOS

**Comment by MarcSabatella** [comment]
I'm not understanding the problem, FWIW.  Articulations are positioned relative to notes, so if you move the note, the articulation moves with it.  An offset of 0 for an articulation means, it is still aligned with the note.  This all seems as it should be.

Fermatas are a different story, they are associated with the time position as a whole, not the specific note.  That's why they don't move when the note moves.  Not saying that's the way it absolutely would have to be be, and in fact, up through MuseScore 2, I think fermatas *were* treated more like articulations.

**Comment by abariska** [comment]
I'd say OK, but it looks really strange when you add appearance offset to selected measure and notes are not synchronised with their articulations. As I understand right, there weren't this problem in MS3 (and, probably, earlier versions) because it wasn't possible to add offset to whole measure when it's selected. So, seems like this case should be handled in some way

**Comment by MarcSabatella** [comment]
Indeed, that wasn't possible to do directly.  Once your selection encompassed multiple types, the Inspector became very limited, only showing properties that really made sense to apply to literally everything in the selection.  In part, to avoid situations like this :-)

I would agree some special handling could be useful for the case of applying offsets with a range selection.  Treating this as if you had only selected notes, rests, and segment-attached elements like fermatas and staff text - skipping elements attached *to* the notes, like articulations of lyrics - is as sensible thing to do as anything I suppose.  Kind of a "do what I mean, not what I say" thing.

**Comment by oktophonie** [comment]
Looking at the original video: what would feel consistent to me is that if _fermatas_, at least, followed their note when the note is offset - just as other 'note-attached' markings do (articulations, bowings, etc.)

Other types of breath marking, caesura, etc, are not directly attached to the note but come at the end of its duration, and therefore should not move when the note is offset.

Then the question of what to do when a whole bar or range is selected and and offset applied is independent.

**Comment by oktophonie** [comment]
(Reopening as only the fermata issue fixed in #19293 - the rest still tbd)
